src/lib/coda_lib/config.ml

```diff
@@ -25,6 +25,7 @@ type t =
   ; work_reassignment_wait: int
   ; gossip_net_params: Gossip_net.Real.Config.t
   ; net_config: Coda_networking.Config.t
+  ; current_fork_id: Fork_id.t
```

Comment:
I suggest renaming this to "initial_fork_id" instead of "current_fork_id". From the CLI, the name "current" makes sense. However, from the config that is passed in to start the node in `coda_lib`, it makes a little less sense since this value is immutable and isn't going to be updated when the current fork id changes dynamically in the future.

Reply:
renamed